webui/src/pages/extension-detail/extension-detail-tabs.tsx

```diff
@@ -34,10 +37,12 @@ export class ExtensionDetailTabsComponent extends React.Component<ExtensionDetai
 
     render(): React.ReactNode {
         const params = this.props.match.params as ExtensionDetailTabs.Params;
-        const tab = params.version === 'reviews' ? 'reviews' : 'overview';
+        const reviewOrChanges = (params.version || '').match(/reviews|changes/);
+        const tab = reviewOrChanges ? params.version : 'overview';
         return <React.Fragment>
             <Tabs value={tab} onChange={this.handleChange}>
                 <Tab value='overview' label='Overview' />
+                <Tab value='changes' label='Changes' />
```

Comment:
Shall we hide the Changes tab in case the changelog is not available? We can check this in advance with the extension metadata.

Reply:
Even if it is not hidden, when the changelog is not available, the Changes tab would still show `No Changelog is available` as content. But, IMO, leaving the tab visible would be better as it encourages extension publisher to include changelog in their extension.
I'm fine either way though (hiding the tab or keeping it visible)... so, I'm leaving this up to you, how do you prefer it?

Reply:
I'm not sure. Let's keep it always visible for now.